geode-core/src/main/java/org/apache/geode/internal/net/SocketCreator.java

```diff
@@ -1032,6 +1038,11 @@ private void configureClientSSLSocket(Socket socket, int timeout) throws IOExcep
       sslSocket.setEnableSessionCreation(true);
 
       if (sslConfig.doEndpointIdentification()) {
+        if (!sslConfig.useDefaultSSLContext()) {
+          logger.warn("Since GEODE 1.7.0, hostname is validated in the Server's certificate "
+              + "during SSL handshake. You can disable using setting '"
+              + SSL_ENDPOINT_IDENTIFICATION_ENABLED + "' to false.");
```

Comment:
The logic and the message here don't match up. You are warning someone if they have intentionally turned on hostname verification.

In any case, I don't think we should be warning people here. The only warning we should give is if they haven't turned on hostname verification, since we will hopefully make that manditory in the future.

Reply:
Thanks @upthewaterspout. Changed to log warning if someone disables it.